src/pages/docs/projects/custom-step-templates.md

```diff
@@ -39,27 +40,21 @@ The Parameters tab allows you to specify fields that will be filled out by the u
 ![Add new step template parameter](/docs/projects/images/step-templates-new-parameter.png)
 :::
 
-You're required to give the parameter a variable name and label to use, as well as some optional help text and a default value.
+You're required to give the parameter a variable name to use, as well as an optional label, help text and default value.
 
-Finally, you can choose the way the field will appear to a user with the **Control type** field. There are a number of options available, however keep in mind the end result will be a variable with a string value.
+You can choose the way the field will appear to a user with the **Control type** field. There are a number of options available, however keep in mind the end result will be a variable with a string value.
 
-:::figure
-![Step template parameter type](/docs/projects/images/step-templates-parameter-type.png)
-:::
-
-**2019.10.0** introduced a new **Package** control type. This new control type allows the selection of a **Feed** and **Package** to use as part of the step to be deferred until the step template is added to a project.
```

Comment:
Removed this + the image as it's difficult to maintain an up-to-date list of available parameter types, and the user should be able to discover this within the dialog themselves.